.github/workflows/reusable_receptorctl_tests.yml

```diff
@@ -0,0 +1,54 @@
+---
+name: receptorctl
+
+"on":
+  workflow_call:
+
+jobs:
+  receptorctl:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        matrix:
+          python-version:
+          # NOTE: The highest and the lowest versions come
+          # NOTE: first as their statuses are most likely to
+          # NOTE: signal problems early
+          - 3.11
+          - 3.8
+          - >-
+            3.10
+          - 3.9
+
+    env:
+      NOXSESSION: tests-${{ matrix.python-version }}
```

Comment:
@webknjaz I see this now. For whatever reason it didn't click from looking at the cherrypy workflow and I was beating myself up trying to get all the sessions to run. Thanks!

Reply:
Yeah, I had to check the nox's docs to see their env var naming scheme.